src/app/public/modules/colorpicker/colorpicker.component.ts

```diff
@@ -76,22 +76,50 @@ import {
 
 let componentIdIndex = 0;
 
+/**
+ * Provides a SKY UX-themed replacement for the HTML `input` element with `type="color"`.
+ * The value that users select is driven through the `ngModel` attribute specified on
+ * the `input` element.
+ */
 @Component({
   selector: 'sky-colorpicker',
   templateUrl: './colorpicker.component.html',
   styleUrls: ['./colorpicker.component.scss']
 })
 
 export class SkyColorpickerComponent implements OnInit, OnDestroy {
+  /**
+   * Fires when users select a color in the colorpicker. Both events
+   * return an object with the following properties:
```

Comment:
Do we need the second sentence here and the code block? The code block applies to both `selectedColorApplied` and `selectedColorChanges`. Do we still need it with all the other stuff docs-tools populates?

Reply:
Good eye; we can remove the second sentence and the code block.